Findcppnetlib.cmake

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright 2015 Grigori Chevtchenko <grigori.chevtchenko@epfl.ch>
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# - Redistributions of source code must retain the above copyright notice, this
+# list of conditions and the following disclaimer.
+# - Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+# - Neither the name of Eyescale Software GmbH nor the names of its
+# contributors may be used to endorse or promote products derived from this
+# software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+#
+# =============================================================================
+#
+# CPPNETLIB 
+# 
+#==============================================================================
+#
+# This module defines the following output variables:
+#
+# CPPNETLIB_FOUND - Was cppnetlib and all of the specified components found?
+#
+# CPPNETLIB_INCLUDE_DIRS - Where to find the headers
+#
+# CPPNETLIB_LIBRARIES - The cppnetlib libraries
+#==============================================================================
+#
+
+# Assume not found.
+SET(CPPNETLIB_FOUND FALSE)
+
+# PATH ________________________________________________________________________
+
+if (NOT CPPNETLIB_PATH)
+    find_path(CPPNETLIB_PATH include/boost/network.hpp
+	HINTS ${CPP_NETLIB_ROOT} $ENV{CPP_NETLIB_ROOT}
```

Comment:
Inconsistency in naming. CMake variable as CPPNETLIB, but the env variable has an underscore in between. I suggest to use the non-underscore one for all variables. Also document the env variable in the comment block above accordingly.